### PR TITLE
refactor: hide unsupported transactions for ledger wallets

### PR DESCRIPTION
**Pull request by luciorubeens**
## Summary

https://app.clickup.com/t/2k1jyu

## Checklist

- [ ] Documentation _(if necessary)_
- [x] Tests _(if necessary)_
- [x] Ready to be merged

### CODE REVIEW

File: src/domains/wallet/pages/WalletDetails/WalletDetails.tsx [hunk 1]
```diff
@@ -77,9 +77,10 @@ export const WalletDetails = ({ txSkeletonRowsLimit, transactionLimit }: WalletD
 	useLayoutEffect(() => {
 		setShowWalletVote(activeWallet.network().can("Transaction.vote"));
 		setShowWalletRegistrations(
-			activeWallet.network().can("Transaction.secondSignature") ||
-				activeWallet.network().can("Transaction.delegateRegistration") ||
-				activeWallet.network().can("Transaction.entityRegistration"),
+			!activeWallet.isLedger() &&
+				(activeWallet.network().can("Transaction.secondSignature") ||
```
Comment (faustbrian) [inline src/domains/wallet/pages/WalletDetails/WalletDetails.tsx:81]: See `1.1.9` and https://github.com/ArkEcosystem/platform-sdk/commit/684bca7fe525efb6fef44cd0c672cc403961b548

**Comment by samharperpittam** [comment]
On this PR I'm still able to attempt to select a Ledger wallet on the Registration page will allow me to attempt to send an AIP36 & second sig transaction. Should we hide these options? I know Simon is working on Multisig support for Ledger but i'm not sure of the status of it (just asked), so depending on how far away it is it might be worth hiding that too for Ledger wallets.

**Comment by luciorubeens** [comment]
@samharperpittam Good catch! Here you go

**Comment by samharperpittam** [comment]
> @samharperpittam Good catch! Here you go

Thank you

**Review by samharperpittam — APPROVED** [review]

**Pull request merged**